Starfish/emulator/emulator.py

```diff
@@ -6,6 +6,7 @@
 import h5py
 import numpy as np
 from nptyping import NDArray
```

Comment:
Just a question... what's the best option for type hints in Python right now? I see [notes in the numpy docs](https://numpy.org/devdocs/reference/typing.html), but it seems like that's 3.8+ only. Is nptyping mostly to support 3.6+, or is there another reason to use the package?

Reply:
At the time I added all the typing (spring/summer 2019) nptyping was the only way to support numpy arrays explicitly. I'll be honest I'm not up to date with the current typing ecosystem, so I'm not sure whats *de facto*.